azure_ad_admin/plugin.spec.yaml

```diff
@@ -387,7 +387,7 @@ actions:
         description: Body of the email to be sent out. Use $password to place the generated password
       notify_recipient:
         required: true
-        title: Recipient of creation email
+        title: Recipient of Creation Email
```

Comment:
Requires a major version bump because the variable change will break any workflows